### PR TITLE
Convert green to orange and index

### PR DESCRIPTION
**Pull request by karst**

### CODE REVIEW

File: resources/g2/sprites.json [hunk 1]
```diff
@@ -18341,13 +18341,13 @@
...
     {
-        "path": "track/bm/brake_horizontal.png",
+        "path": "track/bm_invert/brake_horizontal.png",
         "x_offset": -32,
         "y_offset": 2,
         "palette": "green"
     },
     {
-        "path": "track/bm/brake_vertical.png",
+        "path": "track/bm_invert/brake_vertical.png",
```
Comment (spacek531) [inline resources/g2/sprites.json:18350]: why this change?

**Comment by spacek531** [comment]
This would be a good place to remove the `palette: green` fields since OpenRCT2 does not support that.

File: resources/g2/sprites.json (hunk 1 above)
Comment (karst) [reply resources/g2/sprites.json:18350]: There's already a folder called bm_invert, and since this is a B&M Invert track I am keeping them together.

**Comment by karst** [comment]
> This would be a good place to remove the `palette: green` fields since OpenRCT2 does not support that.

Will do

File: resources/g2/sprites.json (hunk 1 above)
Comment (spacek531) [reply resources/g2/sprites.json:18350]: oh right I forgot that was added
